src/prototype_decryption_encryption/cuban_crow.py

```diff
@@ -19,7 +19,23 @@
 """
 
 
-from typing import Tuple
+import hashlib
+import io
+from pathlib import Path
+from typing import NamedTuple, Optional, Tuple
+
+import crypt4gh.header  # type: ignore
+import crypt4gh.keys  # type: ignore
+
+SRC_DIR = Path(__file__).parent.parent.resolve().absolute()
+FILES_DIR = SRC_DIR.parent.resolve() / "input_files"
```

Comment:
I just called that `INDIR` and the other one `OUTDIR`, but we can decide later when putting everything together